public/app/features/explore/TraceView/TraceView.test.tsx

```diff
@@ -3,13 +3,13 @@ import userEvent from '@testing-library/user-event';
 import { createRef } from 'react';
 import { Provider } from 'react-redux';
 
-import { DataFrame, MutableDataFrame, TimeRange } from '@grafana/data';
+import { DataFrame, TimeRange } from '@grafana/data';
 import { DataSourceSrv, setDataSourceSrv, setPluginLinksHook } from '@grafana/runtime';
 
 import { configureStore } from '../../../store/configureStore';
 
 import { TraceView } from './TraceView';
-import { TraceData, TraceSpanData } from './components/types/trace';
+import { frameOld, frameNew } from './utils/test-mocks';
```

Comment:
Refactored out the mocks because this test would run whenever `public/app/features/explore/TraceView/TraceViewContainer.test.tsx` ran because this test was imported into `public/app/features/explore/TraceView/TraceViewContainer.test.tsx` by the prop `frameOld`

Reply:
FYI I fixed all the other occurrences of this here https://github.com/grafana/grafana/pull/106057 - whichever one goes in second can remove the eslint-disable